bundles/org.eclipse.jdt.doc.user/whatsNew/jdt_whatsnew.html

```diff
@@ -27,18 +27,18 @@ <h2>What's New in Eclipse 4.27 (JDT)</h2>
 They are grouped into:</p>
 <ul> <!-- NOTE: Sync ../topics_WhatsNew.xml with this! -->
   	<!--li><a href="#Java19">Java 19 Support</a></li-->
-	<li><a href="#JavaCompiler">Java Compiler</a></li>
-  	<li><a href="#JUnit">JUnit</a></li>
-	<li><a href="#JavaEditor">Java Editor</a></li>
+	<li><a href="#JavaCompiler" target="_blank">Java Compiler</a></li>
```

Comment:
target="_blank" is added when the link can't open in the same frame and has to be opened in a new tab. It's not required in these places. Any reason it's added?